nullaway/src/main/java/com/uber/nullaway/GenericsChecks.java

```diff
@@ -90,4 +132,141 @@ private static void invalidInstantiationError(
         errorBuilder.createErrorDescription(
             errorMessage, analysis.buildDescription(tree), state, null));
   }
+
+  private static void invalidAssignmentInstantiationError(
+      Tree tree,
+      String lhsAnnotation,
+      String rhsAnnotation,
+      VisitorState state,
+      NullAway analysis) {
+    ErrorBuilder errorBuilder = analysis.getErrorBuilder();
+    ErrorMessage errorMessage =
+        new ErrorMessage(
+            ErrorMessage.MessageTypes.TYPE_PARAMETER_CANNOT_BE_NULLABLE,
```

Comment:
We should create a new message type for this, like `ErrorMessage.MessageTypes.ASSIGN_GENERIC_NULLABLE`